packages/knowledge/src/base.ts

```diff
@@ -678,17 +678,20 @@ export const base: ChartKnowledgeBase = {
   },
 
   boxplot: {
-    id: 'boxplot',
-    name: '',
-    alias: [],
-    family: [],
-    def: '',
-    purpose: [],
-    coord: [],
-    category: [],
-    shape: [],
-    dataPres: [],
-    channel: [],
+    id: 'box_plot',
```

Comment:
the id in https://github.com/antvis/AVA/blob/2fb71a0a4bab9f87ac163961e15ff3302e1f9d10/packages/knowledge/src/chartID.ts#L43 should be exactly 'boxplot', that was because 'boxplot' is already a proper noun.

However, according to https://en.wikipedia.org/wiki/Box_plot it seems **box plot** is more formal than **boxplot**, so I agree with this line. 

But then we should change the line https://github.com/antvis/AVA/blob/2fb71a0a4bab9f87ac163961e15ff3302e1f9d10/packages/knowledge/src/chartID.ts#L43 to 'box_plot'